zboss/doc/zboss_certification.rst

```diff
@@ -25,4 +25,4 @@ The certification guarantees that a platform or product will work and will inter
 Nordic Semiconductor provides the Zigbee Compliant Platform.
 You can use this plaform as the building block for your Zigbee Certified Product, which is conformant with the ZCL and BDB standard.
 
-The latest certification entries for the combination of the ZBOSS v3.3 stack, the |NCS|, and Nordic Semiconductor's SoCs are available on the CSA's `Zigbee Compliant Platforms page`_.
+The latest certification entries for the combination of the ZBOSS v3 stack, the |NCS|, and Nordic Semiconductor's SoCs are available on the CSA's `Zigbee Compliant Platforms page`_.
```

Comment:
v3.0?

Reply:
The second digit increases with nearly every release. If you need it, I propose to write `3.x`

Reply:
Changed to: `v3.3 or newer`